src/main/java/banking/util/CardGenerator.java

```diff
@@ -0,0 +1,42 @@
+package banking.util;
+
+import banking.model.Card;
+
+import java.util.Random;
+
+public class CardGenerator {
+
+    private final CardValidator cardValidator;
+    private final String BINumber;
+
+    public CardGenerator(CardValidator cardValidator, String BINumber) {
+        this.cardValidator = cardValidator;
+        this.BINumber = BINumber;
+    }
+
+    public Card generate() {
+        return new Card(
+                createCardNumber(),
+                createPIN()
+        );
```

Comment:
Depends on your code style preference, but I prefer to place such things at the end of some logical part, not on the new line

Reply:
Oh, okay, thanks for that. Reformatted according to your comment